samples/openapi3/client/petstore/python/petstore_api/configuration.py

```diff
@@ -49,6 +49,19 @@ class Configuration(object):
       then all undeclared properties received by the server are injected into the
       additional properties map. In that case, there are undeclared properties, and
       nothing to discard.
+    :param disable_client_side_validation: Boolean value indicating whether to disable
+      the JSON schema structural validation rules specified in the OpenAPI document.
+      This includes multipleOf, maximum, exclusiveMaximum, minimum, exclusiveMinimum,
+      maxLength, minLength, pattern...
+      By default, the validation is performed for data generated locally by the client
+      and data received from the server, independent of any validation performed by
+      the server side.
+      If the input data does not satisfy the JSON schema validation rules specified
+      in the OpenAPI document, an exception is raised.
+      If disable_client_side_validation is set to true, structural validation is
+      disabled. This can be useful to troubleshoot data validation problem, such as
+      when the OpenAPI document validation rules do not match the actual API data
+      received by the server.
```

Comment:
Is type checking impacted by this change?

Reply:
I'm going to make it a comma-separated list of JSON schema validation keywords. I think this will be more specific and provide more control. Also, that means it can be extends to more keywords.

Reply:
That's a good plan